python/test/unit/mesh/test_higher_order_mesh.py

```diff
@@ -36,10 +36,80 @@ def check_cell_volume(points, cell, domain, volume):
 
         mesh = create_mesh(MPI.COMM_WORLD, [ordered_cell], ordered_points, domain)
         area = assemble_scalar(form(1 * dx(mesh)))
-
         assert np.isclose(area, volume)
 
 
+@pytest.mark.skip_in_parallel
+@pytest.mark.parametrize('order', range(1, 5))
+def test_submesh(order):
```

Comment:
This test is hard to follow. Explanation on what is being done would be helpful.

Reply:
The mesh generator is the same as we use everywhere else in the `test_higher_order_mesh.py`.